From 31653832d72ce7dc6203cb10133ba94b05880dab Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Mon, 5 Oct 2015 05:17:54 -0400
Subject: [PATCH 1/2] Set releasever

...so that $releasever is properly substituted in the kickstart repository
locations.
---
 appcreate/appliance.py  | 4 ++--
 tools/appliance-creator | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/appcreate/appliance.py b/appcreate/appliance.py
index 3afe993..f77b13b 100644
--- a/appcreate/appliance.py
+++ b/appcreate/appliance.py
@@ -43,13 +43,13 @@ class ApplianceImageCreator(ImageCreator):
 
     """
 
-    def __init__(self, ks, name, disk_format, vmem, vcpu):
+    def __init__(self, ks, name, disk_format, vmem, vcpu, releasever=None):
         """Initialize a ApplianceImageCreator instance.
 
         This method takes the same arguments as ImageCreator.__init__()
 
         """
-        ImageCreator.__init__(self, ks, name)
+        ImageCreator.__init__(self, ks, name, releasever=releasever)
 
         self.__instloop = None
         self.__imgdir = None
diff --git a/tools/appliance-creator b/tools/appliance-creator
index 1708431..9e3fa15 100755
--- a/tools/appliance-creator
+++ b/tools/appliance-creator
@@ -129,7 +129,7 @@ def main():
     if options.name:
         name = options.name
             
-    creator = appcreate.ApplianceImageCreator(ks, name, options.disk_format, options.vmem, options.vcpu)
+    creator = appcreate.ApplianceImageCreator(ks, name, options.disk_format, options.vmem, options.vcpu, releasever=options.version)
     creator.tmpdir = options.tmpdir
     creator.checksum = options.checksum
 
-- 
2.9.3

